exporter/splunkhecexporter/factory.go

```diff
@@ -87,6 +87,12 @@ func createDefaultConfig() component.Config {
 		},
 		HealthPath:            splunk.DefaultHealthPath,
 		HecHealthCheckEnabled: false,
+		WorkerQueueConfig: WorkerQueueConfig{
```

Comment:
this config change needs discussion and should be done in a separate PR so we can discuss its merits.